io.qt.PySide.BaseApp.yaml

```diff
@@ -13,13 +13,15 @@ build-options:
   append-ld-library-path: /usr/lib/sdk/llvm16/lib
   env:
     LLVM_INSTALL_DIR: /usr/lib/sdk/llvm16
+    SHIBOKEN_DEBUG_LEVEL: 'full'
 modules:
   - python3-requirements.json
   - name: pyside-setup
     buildsystem: simple
     build-commands:
       - python3 setup.py build --qtpaths=/usr/bin/qtpaths --ignore-git --parallel=8
-        --log-level=verbose --no-qt-tools --shiboken-force-process-system-headers
+        --log-level=verbose --no-qt-tools --flatpak
+      - cat /run/build/pyside-setup/build/qfp-py3.11-qt6.7.2-64bit-release/build/pyside6/PySide6/QtPdf/mjb_rejected_classes.log
```

Comment:
Can this be calculated? the moment we update the Python or Qt version we will need to update this manually.

Reply:
come to think of it, this line is completely useless and can be removed. I added it for debugging purposes when i tested it.